consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -87,33 +87,36 @@ func TestCombinedSignWithDKGKey(t *testing.T) {
 	vote, err = signer.CreateVote(block)
 	require.NoError(t, err)
 
-	voteValid, err := verifier.VerifyVote(nodeID, vote.SigData, block)
+	err = verifier.VerifyVote(nodeID, vote.SigData, block)
 	require.NoError(t, err)
-	require.Equal(t, true, voteValid)
 
 	// vote on different bock should be invalid
 	blockWrongID := *block
 	blockWrongID.BlockID[0]++
-	_, err = verifier.VerifyVote(nodeID, vote.SigData, &blockWrongID)
+	err = verifier.VerifyVote(nodeID, vote.SigData, &blockWrongID)
 	require.Error(t, err)
+	require.True(t, errors.Is(err, model.ErrInvalidSignature))
```

Comment:
you can simplify it by using `require.ErrorIs`

Reply:
line https://github.com/onflow/flow-go/blob/344e69f50ea374cabac8abd5ab84a179f51baa69/consensus/hotstuff/verification/combined_signer_v2_test.go#L97 is not required as `errors.Is( .. )` already fails for `nil`